docs/release-notes/december-2023.md

```diff
@@ -0,0 +1,145 @@
+---
+slug: december-2023
+append_help_link: true
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: December 2023
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for December 2023
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in December 2023:
+
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.52.0](https://github.com/returntocorp/semgrep/releases/tag/v1.52.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.53.0](https://github.com/returntocorp/semgrep/releases/tag/v1.53.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.0](https://github.com/returntocorp/semgrep/releases/tag/v1.54.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.1](https://github.com/returntocorp/semgrep/releases/tag/v1.54.1)
+
+## 🌐 Cloud Platform
+
+### Added
+
+* Semgrep IDE integrations now cache information about the current repository so
+  that it doesn't traverse the entire repository on every scan to determine if
+  the files are valid targets for scanning; this improves scan times.
+* Users can now ignore findings locally in Semgrep IDE Extensions. The changes
```

Comment:
```suggestion
* Users can now ignore findings locally in Semgrep IDE extensions. The changes
```
;_; Does it keep eating my corrections?

Reply:
I think so; I double-checked this was there, too!